source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,188 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having to 
+manage your own Kafka cluster. 
+
+Before you Begin
+----------------
+
+Due to restrictions by Confluent, you are only able to use Basic, Standard, and 
+Dedicated clusters. For most use cases, the basic and standard cluster are enough 
+to handle migrating large volumes of data.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal 
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
```

Comment:
**Suggestion**

Not sure if we want to actively advertise for CC, but it might be worth a mention that you can get started without a paywall / credit card on the free tier /30 days?

Reply:
Going to look into seeing if we can add this!